LICENSE.txt

```diff
@@ -73,6 +76,9 @@ SmallJoker:
   textures/base/pack/cdb_clear.png
   textures/base/pack/server_favorite_delete.png (based on server_favorite.png)
 
+DS:
+  games/devtest/mods/soundstuff/textures/soundstuff_bigfoot.png
```

Comment:
I haven't found any information of authors of devtest media files elsewhere. So, I'm not sure if this the best place.

As such, c55 is given as author:
>Everything not listed in here:
>Copyright (C) 2010-2012 celeron55, Perttu Ahola <celeron55@gmail.com>

cc @Wuzzy2